typescript/index.js

```diff
@@ -0,0 +1,75 @@
+var Package = require('dgeni').Package;
+var path = require('canonical-path');
+
+// Define the dgeni package for generating the docs
+module.exports = new Package('typescript', [require('../base')])
+
+// Register the services and file readers
+.factory(require('./services/modules'))
+.factory(require('./services/tsParser'))
+.factory(require('./services/tsParser/createCompilerHost'))
+.factory(require('./services/tsParser/getFileInfo'))
+.factory(require('./services/tsParser/getExportDocType'))
+.factory(require('./services/tsParser/getContent'))
+
+.factory(require('./services/convertPrivateClassesToInterfaces'))
+
+.factory('EXPORT_DOC_TYPES', function() {
+  return [
+    'class',
+    'interface',
+    'function',
+    'var',
+    'const',
+    'let',
+    'enum',
+    'type-alias'
+  ];
+})
+
+// use `untouchedNamespaces.push()` to inject your own regexes here.
+// these namespaces will not be stripped if used for type parameters.
+.factory('untouchedNamespaces', function() { return []; })
```

Comment:
I just looked and there are no longer any references to `rxNext` in the Angular 2 source so I think that this is actually no longer a big deal. But fine to leave in anyway.